drafts/FPAAnnualReport2022.md

```diff
@@ -14,72 +14,59 @@ The administrators and members of the FPA would like to thank all of you who put
 
 In 2022, the FPA was busy preparing for the [2023 Free and Open Source Developers’ European Meeting (FOSDEM)](https://fosdem.org).  Arrangements for a [FreeCAD meeting day](https://fpa.freecad.org/programs/freecad-day-2023) on the Friday immediately preceeding FOSDEM began late in 2022.
 
 A [grant program](https://fpa.freecad.org/programs/fosdem-travel-grants) was established to help defray travel costs for attendees.
 
-
 ### Branding
 
 The FPA acquired control of the [freecad.org](https://freecad.org) domain, which was generously donated to the FreeCAD community by the [KiCAD Services Corp.](https://www.kipro-pcb.com/). The FPA also registered a FreeCAD [trademark](https://fpa.freecad.org/trademark.pdf).  These steps protect the FreeCAD brand.
 
-
 ### New members
 
 The FPA [roster](https://fpa.freecad.org/handbook/people/roster.html) has expanded. More people have joined the FPA, and people have started assuming different roles and positions. So far the arrival of new members has been spontaneous, without much regulation or direction, but as the FPA grows this is a question that is under discussion.
 
-
 ### Administration team
 
 The FPA administration team expanded to four members in 2022 and is now working at full power. The team maintains a weekly video meeting and no week passes without several tasks being done. Everything learned or set up is being documented in the [FPA handbook](https://fpa.freecad.org/handbook). The handbook covers everything about how the FPA is organized and operated, and serves as a reference for people both inside and outside the FPA.
 
-
 ## Plans for 2023
 
 We expect 2023 to be even busier than 2022.  Some of the things we hope to accomplish in 2023 are:
 
-* Enable the FPA to engage in commercial operations and deduct VAT
-
-* Hire an accountant to handle Belgian reporting requirements, such as tax returns
-
-* Based on 2022 donations, our expected income is €75,000
+* **Enable the FPA to engage in commercial operations** and deduct VAT. The FPA has up to now been exempt from VAT registration, because it operates only on donations. In 2023, we want to enable the FPA to sign contracts and sell services, so it can act as an intermediary and make it easier for companies to contract FreeCAD developers. The VAT registration will enable the FPA to do so, and provide it with a VAT number, which allows it to pay less taxes when buying services and products in the EU.
 
-* Expand the FPADF grant program to a larger amount (€2,000 or €2,500)
+* **Hire an accountant** to handle Belgian reporting requirements, such as tax returns and VAT handling. Up to now, the FPA members have done all the accounting tasks themselves. Hiring an external accountant, however, will guarantee that the FPA does no accounting or tax errors, and will allow the FPA to keep functioning even in case of large changes among FPA members.
```

Comment:
```suggestion
* **Hire an accountant** to handle Belgian reporting requirements, such as tax returns and VAT handling. Up to now, the FPA members have done all the accounting tasks themselves. Hiring an external accountant, however, will help to ensure that the FPA makes no accounting or tax errors, and will allow the FPA to keep functioning even in case of large changes among FPA members.
```